experimental/LieAlgebras/src/LinearLieAlgebra.jl

```diff
@@ -140,21 +140,17 @@ end
 ###############################################################################
 
 @doc raw"""
-    (L::LinearLieAlgebra{C})(m::MatElem{C}) -> LieAlgebraElem{C}
+    coerce_to_lie_algebra_elem(L::LinearLieAlgebra{C}, x::MatElem{C}) -> LinearLieAlgebraElem{C}
```

Comment:
Hmm, so `L(m)` is changed to `coerce_to_lie_algebra_elem(L,m)` -- why?

Reply:
Because `L(m)` is already in use for `m` being a coefficient vector (of size `1 x dim(L)`). For dimensions 1 and 0, it is ambiguous which of the two you mean

Reply:
Why are coefficient vectors matrices and not vectors?

Reply:
That's because I started with `AbstractAlgebra`'s `Generic.FreeModuleElem` and adapted it to my needs.

I should probably change it to `SRow{C}` and change module action matrices and home matrices to sparse matrices as well. But as that is a larger change, I would like to split that into a new PR.
`coerce_to_lie_algebra_elem(L, m)` will then be removed again in favor of `L(m)`.